File: problems/problems_837/solution.go
package problem837

import (
	"encoding/json"
	"log"
	"strings"
)

func new21Game(n int, k int, maxPts int) float64 {
    
}

func Solve(inputJsonValues string) any {
	inputValues := strings.Split(inputJsonValues, "\n")
	var n int
	var k int
	var maxPts int

	if err := json.Unmarshal([]byte(inputValues[0]), &n); err != nil {
		log.Fatal(err)
	}
	if err := json.Unmarshal([]byte(inputValues[1]), &k); err != nil {
		log.Fatal(err)
	}
	if err := json.Unmarshal([]byte(inputValues[2]), &maxPts); err != nil {
		log.Fatal(err)
	}

	return new21Game(n, k, maxPts)
}

File: problems/problems_837/solution.ts
function new21Game(n: number, k: number, maxPts: number): number {
    
};

export function Solve(inputJsonElement: string): any {
	const inputValues: string[] = inputJsonElement.split("\n");
	const n: number = JSON.parse(inputValues[0]);
	const k: number = JSON.parse(inputValues[1]);
	const maxPts: number = JSON.parse(inputValues[2]);
	return new21Game(n, k, maxPts);
}
